src/main/scala/mesosphere/marathon/DebugConf.scala

```diff
@@ -11,26 +11,22 @@ import org.rogach.scallop.ScallopConf
 
 /**
   * Options related to debugging marathon.
-  * All options should be optional and turned off by default.
   */
 trait DebugConf extends ScallopConf {
 
-  lazy val debugTracing = opt[Boolean]("enable_tracing",
```

Comment:
keep this as a hidden option to ignore the comment and allow Marathon to startup